Client Scripts/validate phone number/validateLength.js

```diff
@@ -0,0 +1,13 @@
+// Client Script: Validate Length
+// Table: sys_user
+// Type: onChange
+// Field: phone
+
+function onChange(control, oldValue, newValue, isLoading) {
+    if (isLoading || newValue === '') return;
+
+  if (newValue.length>12) {
```

Comment:
What if somebody uses spaces or dashes in the phone number? `+1 (800) 123-4567` is 17 characters and is a valid format for a phone number in the US. Have you considered using a regex to test the phone number?  Additionally there is a dedicated field type for phone numbers, so can you provide some context around when you would use this script instead of the field?